presto-main/src/test/java/com/facebook/presto/memory/TestMemoryPools.java

```diff
@@ -78,9 +78,7 @@ public void testBlocking()
         }
 
         // driver should be blocked waiting for memory
-        for (Driver driver : drivers) {
-            assertFalse(driver.isFinished());
-        }
+        assertTrue(isWaitingForMemory(drivers));
```

Comment:
As local exchange now buffers data to produce full pages drivers with TableScan -> LocalExchangeSink is not always instantly blocked by the drivers above.